Various updates. (#5273)
flutter/website: src/web.md

  If you experience a problem that hasn't yet been reported, please
  [file an issue][] and make sure that "web" appears in the title.
{{site.alert.end}}

{{site.alert.warning}}

>   **Hot reload not still supported**

Reviewer: ```suggestion
  **Hot reload not supported**
```

>   **Hot reload not supported**

Reviewer: ```suggestion
  **Hot reload not supported:**
```

>   At of beta, web supports only **hot restart**;

Reviewer: ```suggestion
  As of beta, only **hot restart** is supported on the web
```

>   As of beta, only **hot restart** is supported on the web

Reviewer: ```suggestion
  As of beta, only **hot restart** is supported on the web;
```
